docs/development/maintenance.md

```diff
@@ -13,1292 +15,1461 @@ modules of ILIAS.
+    * Unit-specific Guidelines, Rules, and Regulations: [LINK MISSING]('')
 
 [//]: # (END OpenIdConect)
 
 [//]: # (BEGIN OrganisationalUnits)
 
 * **Organisational Units**
-	* Maintenance Model: Classic
-	* 1st Maintainer: [rklees](https://docu.ilias.de/goto_docu_usr_34047.html)
-	* 2nd Maintainer: [fschmid](https://docu.ilias.de/goto_docu_usr_21087.html)
-	* Testcases: [wischniak](https://docu.ilias.de/goto_docu_usr_21896.html)
+    * Authority to Sign off Code Changes: [rklees](https://docu.ilias.de/goto_docu_usr_34047.html)
+        , [fschmid](https://docu.ilias.de/goto_docu_usr_21087.html)
+    * Authority to Sign off Conceptual Changes: [rklees](https://docu.ilias.de/goto_docu_usr_34047.html)
+    * Authority to Sign off Testcase Changes: [wischniak](https://docu.ilias.de/goto_docu_usr_21896.html)
+    * Authority to (De-)Assign Authoritiees: [rklees](https://docu.ilias.de/goto_docu_usr_34047.html)
 	* Tester: [qualitus.morgunova](https://docu.ilias.de/goto_docu_usr_69410.html)
+    * Unit-specific Guidelines, Rules, and Regulations: [LINK MISSING]('')
 
 [//]: # (END OrganisationalUnits)
 
 [//]: # (BEGIN PDF)
 
 * **PDF**
-	* Maintenance Model: Classic
-	* 1st Maintainer: [gvollbach](https://docu.ilias.de/goto_docu_usr_25234.html)
-	* 2nd Maintainer: N.A.
-	* Testcases: [AUTHOR MISSING](https://docu.ilias.de/goto_docu_pg_64423_4793.html)
+    * Authority to Sign off Code Changes: [gvollbach](https://docu.ilias.de/goto_docu_usr_25234.html)
+    * Authority to Sign off Conceptual Changes: [gvollbach](https://docu.ilias.de/goto_docu_usr_25234.html)
+    * Authority to Sign off Testcase Changes: [gvollbach](https://docu.ilias.de/goto_docu_usr_25234.html)
+    * Authority to (De-)Assign Authoritiees: [gvollbach](https://docu.ilias.de/goto_docu_usr_25234.html)
 	* Tester: [TESTER MISSING](https://docu.ilias.de/goto_docu_pg_64423_4793.html)
+    * Unit-specific Guidelines, Rules, and Regulations: [LINK MISSING]('')
 
 [//]: # (END PDF)
 
 [//]: # (BEGIN PersonalAndSharedResources)
 
 * **Personal and Shared Resources**
-	* Maintenance Model: Classic
-	* 1st Maintainer: [akill](https://docu.ilias.de/goto_docu_usr_149.html)
-	* 2nd Maintainer: N.A.
-	* Testcases: [AUTHOR MISSING](https://docu.ilias.de/goto_docu_pg_64423_4793.html)
+    * Authority to Sign off Code Changes: [akill](https://docu.ilias.de/goto_docu_usr_149.html)
+    * Authority to Sign off Conceptual Changes: [akill](https://docu.ilias.de/goto_docu_usr_149.html)
+    * Authority to Sign off Testcase Changes: [akill](https://docu.ilias.de/goto_docu_usr_149.html)
+    * Authority to (De-)Assign Authoritiees: [akill](https://docu.ilias.de/goto_docu_usr_149.html)
 	* Tester: [scarlino](https://docu.ilias.de/goto_docu_usr_56074.html)
+    * Unit-specific Guidelines, Rules, and Regulations: [LINK MISSING]('')
 
 [//]: # (END PersonalAndSharedResources)
 
-[//]: # (BEGIN Poll)
+[//]: # (BEGIN PersonalProfile)
```

Comment:
Hi @kergomard, I think this should still read `BEGIN Poll`

Reply:
Thank you very much @schmitz-ilias ! I've fixed it.